Apps/W1/HybridBaseDeployment/app/src/tables/MigrationValidationBuffer.Table.al

```diff
@@ -0,0 +1,111 @@
+namespace Microsoft.DataMigration;
+
+table 40046 "Migration Validation Buffer"
```

Comment:
This table is not the best approach. We need a different approach to store the data that needs to be validated.